README.md

```diff
@@ -57,6 +57,11 @@ Use this option to set the series data for your chart.
 The `theme` argument is optional and it allows you to pass in a
 [Highcharts theme](http://www.highcharts.com/docs/chart-design-and-style/themes).
 
+### callback
+
+The `callback` argument is executed after the chart object is finished loading and rendering.
```

Comment:
Let's mention that its an optional parameter:

The `callback` argument is optional and is executed after the chart object is finished loading and rendering.